### ISSUE TITLE
[TOPIC] Misc other things to write about 

### ISSUE THREAD

**Issue by Carreau** [jupyter/newsletter#4]
Various things to triage, potentially make a link and a short description. 
## https://github.com/bollwyvl/nb-mermaid (diagram in IPython notebook)

For those of you who work a lot on graphs, `@bollwyvl` have written an IPython extension to easily create [mermaid] graphs. Head up to [his repository] to know more and see the examples. 
## Convert notebooks from Python 2 to Python 3.

Python 3 is reaching end of life in 2020, it is more that time to update your notebooks ! Using the last version of nbconvert, and this package, you can know run [2to3] on your notebook to update them to Python 3 ! Warning the tool is still in early developpement stage. [note, we need to wait for a release of nbconvert for this to work]

[`jupyter nbconvert --to 2to3 <notebook>`](https://pypi.python.org/pypi/jupytercontrib#to3-for-notebooks), maybe a bit young
## Create better slideshows  with notebook

Developpers at Continuum Analytics continue to create Amazing extensions ! Here is a [nbpresent], a hat allow you to create beautiful slideshow from your notebook ! Head there to try the lastest version and give them your feedback. 

https://github.com/Anaconda-Server/nbpresent (better slideshow for Notebook)
## PyData Amsterdam

Thomas and Min will be at PyData Amsterdam:  12th/13th March. Min & Thomas will talk about multi-language support and new/upcoming features

Can we link to talks ?
## Jdays Boston

JupyterDays Boston (already on the blog though) cf other issue
## Notebook ScratchPad

Do you keep experimenting in your notebooks, and deleting your experimentation cells ? Notebook Scratch pad is made for you. Thanks to `@minrk`, you can know with a single keypress get a temporary cell in the notebook to write one-of commands !

Min's scratchpad extension: https://github.com/minrk/nbextension-scratchpad
## Think Bayes

Think Bayes by Allen B. Downey is a well known O'Reilly  book that introduce reader to Bayesian probability. thanks to `@rlabbe` is is now available on Binder; you can thus run the all book online and explore by yourself the code example !

https://github.com/rlabbe/ThinkBayes as Jupyter notebook.
## Interactive blog post.

Jupyter provide interactivity and live code execution well beyond the notebook. In one of his  post, `@betatim` explore how to make its blog interactive, and allowing the reader to execute code dirrectly inline !

Make interactive blog posts backed by a Jupyter kernel: http://betatim.github.io/posts/really-interactive-posts/

Links to JupyterDayChicago Talks, in for example how to create a pugme extension https://github.com/mdagost/pugme-jupyter

**Comment by katiewhite360** [comment]
> https://github.com/bollwyvl/nb-mermaid (diagram in IPython notebook)
> 
> jupyter nbconvert --to 2to3 <notebook>, maybe a bit young
> 
> https://github.com/Anaconda-Server/nbpresent (better slideshow for Notebook)
> 
> Min's scratchpad extension: https://github.com/minrk/nbextension-scratchpad
> 
> https://github.com/rlabbe/ThinkBayes as Jupyter notebook.
> 
> Make interactive blog posts backed by a Jupyter kernel: http://betatim.github.io/posts/really-interactive-posts/

Matthias, if you could pick one, which of these ^^ would be worth a write up this week? Or possibly links to a couple?

**Comment by Carreau** [comment]
Probalby the  last one, it's olready an old thing. 
I'll try to writeup something.

**Comment by Carreau** [comment]
@katiewhite360 I've updated above text to provide a (short) blurb that describe each link. 
You should have enough permissions to actually edit my post yourself, if you want to refine.

**Comment by katiewhite360** [comment]
Does anyone have the links to Thomas and Min's Amsterdam talks?

**Comment by willingc** [comment]
@takluyver @minrk ^^ for @katiewhite360. Thanks.

**Comment by takluyver** [comment]
Voila: https://www.youtube.com/watch?v=T385txAYSt8

**Comment by Ruv7** [comment]
@Carreau  @katiewhite360 - I'm closing out open issues that have exhausted their use. Anything worth posting about in future newsletters in here or should I close this out?

**Comment by Ruv7** [comment]
Closing this issue out. Here is the process I believe we settled on: To submit a content suggestion for upcoming Jupyter newsletter one should:

Create a new issue here: https://github.com/jupyter/newsletter/issues - follow the format of the template post, and label your issue "topic suggestion". Content suggestions at a minimum should include a headline and 2 -4 sentences that are ready to publish. Where possible a link to where the reader can get more information and/or a picture associated with the story should be included.
